bam_paired.cwl

```diff
@@ -33,6 +33,25 @@ inputs:
     type: string?
   - id: column_number
     type: int?
+  - id: alignEndsType
```

Comment:
should these optional ones be added here if they're only going to be used for miRNA?